frontend/services/label/getters.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const getLabels = () => {
+  return axios.get(`http://localhost:3000/api/labels`);
+};
+
+const getters = () => {
+  return {
+    getLabels,
+  };
+};
+
+export default getters;
```

Comment:
label 관련 api들을 모아서 객체로 export하려면 이렇게는 어떨까요? 
```suggestion
import axios from 'axios';

const getLabels = () => { ... }
const addLabel = () => { ... }
const deleteLabel = () => { ... }
const editLabel = () => { ... }

export { getLabels, addLabel, deleteLabel, editLabel };
```